src/components/Tokens/TokenDetails/InfoTip.tsx

```diff
@@ -0,0 +1,34 @@
+import Tooltip from 'components/Tooltip'
+import { ReactNode, useCallback, useState } from 'react'
+import { Info } from 'react-feather'
+import styled from 'styled-components/macro'
+
+const InfoTipContainer = styled.div`
+  display: flex;
+  position: relative;
+  align-items: center;
+  cursor: pointer;
```

Comment:
nit: (don't think we do this elsewhere) - since it's not clickable, we prob should do `cursor: help` instead of `pointer`